Assets/MirageSDK/Plugins/WalletConnectSharp.Unity/Network/Client/EventHandlers/WebSocketCloseEventHandler.cs

```diff
@@ -0,0 +1,6 @@
+using MirageSDK.WalletConnectSharp.Unity.Network.Client.Data;
+
+namespace MirageSDK.WalletConnectSharp.Unity.Network.Client.EventHandlers
```

Comment:
Why do we have a new files?

Reply:
hmmm maybe a walletconnect auto update that added that ?